presto-docs/src/main/sphinx/release/release-0.287.rst

```diff
@@ -0,0 +1,98 @@
+* Add support for ``map_key_exists`` builtin SQL UDF. :pr:`21966`
+* Optimize ``map_normalize`` builtin SQL UDF to avoid repeated reduce computation. :pr:`22211`
+* Add configuration property ``legacy_json_cast`` whose default value is ``true``. See `Legacy Compatible Properties <../admin/properties.html#legacy-compatible-properties>`_. :pr:`21869`
+* Add support for tracking of the input data size when there is a fragment result cache hit. This can be enabled by setting the configuration property ``fragment-result-cache.input-data-stats-enabled=true``. :pr:`22145`
+* Remove ``native_execution_enabled``, ``native_execution_executable_path`` and ``native_execution_program_arguments`` session properties. Corresponding configuration properties are still available. :pr:`22183`
+* Remove the configuration property ``use-legacy-scheduler`` and the corresponding session property ``use_legacy_scheduler``.   The property previously defaulted to true, and the new scheduler, which was intended to replace it eventually, was never productionized and is no longer needed. The configuration property ``max-stage-retries`` and the session property ``max_stage_retries`` have also been removed. :pr:`21952`
+* Add JSON as a supported output format in the Presto CLI. :pr:`22181`
+* Upgrade Alluxio to 310. :pr:`22012`
+* Upgrade Alluxio to 312. :pr:`22452`
+* Upgrade Airlift version to 0.209. :pr:`21943`
+* Add documentation for supported data `Type mapping <../connector/iceberg.html#type-mapping>`_  in the Iceberg connector. :pr:`22093`
+* Add usage documentation for :doc:`/clients/presto-cli`. :pr:`22265`
+* Add usage documentation for :doc:`/clients/presto-console`. :pr:`22349`
+* Add :doc:`Prestissimo Developer Guide <../prestissimo>` topic to the Presto documentation. :pr:`21953`
+
+Security Changes
+________________
+* Remove logback 1.2.3. :pr:`21819`
+* Add session property ``default-view-security-mode`` to choose the default security mode for view creation. :pr:`21956`
+
+Verifier Changes
+________________
+* Add support for extended bucket verification of INSERT and CTAS queries. This can be enabled by the configuration property ``extended-verification`` to verify each bucket's data checksum if the inserted table is bucketed. :pr:`22001`
+* Add support for extended partition verification of INSERT and CTAS queries. This can be enabled by the configuration property ``extended-verification`` to verify each partition's data checksum if the inserted table is partitioned. :pr:`21983`
+
+SPI Changes
+___________
+* Add replaceColumn method to com.facebook.common.Page. :pr:`22493`
+* Remove SPI method ConnectorMetadata.getTableLayouts() as deprecated. Add ConnectorMetadata.getTableLayoutForConstraint() as replacement. :pr:`21933`
+* Move `SortNode` to SPI module to be utilized in connector. :pr:`22497`
+
+Hive Changes
```

Comment:
```suggestion
Hive Connector Changes
```
Following the [Section naming](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#section-naming) guidelines in Release Notes Guidelines.